internal/controller/driver_controller.go

```diff
@@ -756,6 +756,32 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 								),
 							})
 						}
+						// ODF Snapshotter Sidecar Container
+						if r.driverType != NfsDriverType && snPolicy == csiv1a1.VolumeGroupSnapshotPolicy {
```

Comment:
we might need a extra check using ENV to enable/disable this feature, lets not use `snPolicy == csiv1a1.VolumeGroupSnapshotPolicy` check here

Reply:
Should we have the env variable added only in the CSV or all the type of deployment specs that we have for the operator like, all-in-one, helm, multifile?
If it's all then I can edit the variable `BUILD_INSTALLER_OVERLAY` in the Makefile to add the env or else, I will have to create a different variable that we would use while generating the bundle which would have the env variable.

Reply:
Done, PTAL.